graylog2-web-interface/src/views/components/WidgetQueryControls.test.jsx

```diff
@@ -118,11 +118,11 @@ describe('WidgetQueryControls', () => {
       expect(queryByText(indicatorText)).toBeNull();
     });
 
-    it('disables timerange controls when global override is present', () => {
-      const { getByDisplayValue } = renderSUT({ globalOverride: globalOverrideWithQuery });
-      const timeRangeSelect = getByDisplayValue('Search in last day');
-
-      expect(timeRangeSelect).toBeDisabled();
-    });
+    // it('disables timerange controls when global override is present', () => {
```

Comment:
All of these commented out tests are for reference later to make sure any test removed is properly replaced before the feature branch is merged to master later